pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright 2016 Inscope Metrics Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.arpnetworking.build</groupId>
+    <artifactId>arpnetworking-parent-pom</artifactId>
+    <version>1.0.29</version>
+    <relativePath />
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.arpnetworking.metrics.extras</groupId>
+  <artifactId>incubator-extra</artifactId>
+  <packaging>jar</packaging>
+  <name>Metrics client incubator</name>
+  <description>Incubator for features that will later be built into the core metrics client.</description>
+  <url>https://github.com/ArpNetworking/metrics-incubator-extra</url>
+  <version>0.6.0-SNAPSHOT</version>
```

Comment:
Because it matches up with the other versions fairly closely.  It it built against 0.6.X of the metrics client.  I don't feel strongly about the versioning and will change if you want.